lucene/core/src/java/org/apache/lucene/codecs/VectorWriter.java

```diff
@@ -153,13 +153,12 @@ public int nextDoc() throws IOException {
     private final DocIDMerger<VectorValuesSub> docIdMerger;
     private final int[] ordBase;
     private final int cost;
-    private final int size;
+    private int size;
 
     private int docId;
     private VectorValuesSub current;
-    // For each doc with a vector, record its ord in the segments being merged. This enables random
-    // access into the
-    // unmerged segments using the ords from the merged segment.
+    /* For each doc with a vector, record its ord in the segments being merged. This enables random access into the unmerged segments using the ords from the merged segment.
+     */
```

Comment:
Hmmm I thought spotless would wrap this line, but it doesn't seem to complain about it